multiqc_clarity/multiqc_clarity.py

```diff
@@ -50,9 +56,14 @@ def __init__(self):
 
 
     def get_samples(self):
-        if config.kwargs.get('clarity_project_name'):
```

Comment:
Do you mean to remove this?

Reply:
Yes, because the option name is actually `clarity_project`, and not `clarity_project_name`.

Reply:
Ah, maybe I'm confused. In `setup.py` it's still `clarity_project_name`... But maybe that doesn't matter..?

https://github.com/MultiQC/MultiQC_Clarity/blob/d55ae4fddc3d819d4a65d8d6b41adbf3957d1501/setup.py#L37

Reply:
I think it is based on the name which is specified in the click option. However, to make it clearer I could rename the variable to have the same name?

Reply:
Yeah I think that would be a good idea if that's ok. Was this not working before then?

Reply:
Nope. It was not working. I'll fix the variable name.